reference/JsGetPromiseResult.md

```diff
@@ -0,0 +1,21 @@
+Gets the result of a given promise object.
+
+### Syntax 
+```
+CHAKRA_API
+    JsGetPromiseResult(
+        _In_ JsValueRef *promise,
+        _Out_ JsPromiseState *statusCode);
```

Comment:
Looks like you've just copy/pasted from the other method; this should be a `JsValueRef` out parameter I believe.